docs/book/intro.md

```diff
@@ -8,7 +8,7 @@ if a user correctly provides the first word, the second is likely correctly ente
 improve OCR software for digitizing books.
 
 <!-- markdownlint-disable-next-line header-increment -->
-> ### Additional Requirements
+> ## Additional Requirements
```

Comment:
```suggestion
> ### Additional Requirements
```

Reply:
@gsteel This will result in failure of QA Checks (README Linting)

Reply:
@manishranjan-adobe 
No, the rule is disabled. See: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md001---heading-levels-should-only-increment-by-one-level-at-a-time

Reply:
@froschdesign I have corrected the piece of the line. cheers